apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -7,23 +7,25 @@ import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
 import {setCurrentAiModalField, DanceState} from '../danceRedux';
 import classNames from 'classnames';
 import {FieldDropdown, Workspace} from 'blockly/core';
-import AiGeneratingView from './AiGeneratingView';
 import {chooseEffects} from './DanceAiClient';
 import AiVisualizationPreview from './AiVisualizationPreview';
 import AiBlockPreview from './AiBlockPreview';
 import AiExplanationView from './AiExplanationView';
-import {AiOutput, FieldKey} from '../types';
+import {AiOutput, FieldKey, GeneratedEffect, Scores} from '../types';
 import {generateBlocks, generateBlocksFromResult, getLabelMap} from './utils';
+const ToggleGroup = require('@cdo/apps/templates/ToggleGroup').default;
+import color from '@cdo/apps/util/color';
 
-const aiBotBorder = require('@cdo/static/dance/ai/ai-bot-border.png');
-const aiBotBeam = require('@cdo/static/dance/ai/blue-scanner.png');
+const aiBotBorder = require('@cdo/static/dance/ai/bot/ai-bot-border.png');
+const aiBotYes = require('@cdo/static/dance/ai/bot/ai-bot-yes.png');
+const aiBotNo = require('@cdo/static/dance/ai/bot/ai-bot-no.png');
+const aiBotBeam = require('@cdo/static/dance/ai/bot/blue-scanner.png');
```

Comment:
I just merged https://github.com/code-dot-org/code-dot-org/pull/54520, so you should be able to `import` these directly now instead of needing to use `require` 🙂

Reply:
Oh whoops ignore, this was already taken care of!